edc-extensions/observability-api-customization/src/test/java/org/eclipse/tractusx/edc/api/observability/TxObservabilityApiControllerTest.java

```diff
@@ -0,0 +1,136 @@
+package org.eclipse.tractusx.edc.api.observability;
+
+import io.restassured.specification.RequestSpecification;
+import org.eclipse.edc.connector.spi.asset.AssetService;
+import org.eclipse.edc.connector.spi.catalog.CatalogService;
+import org.eclipse.edc.connector.spi.contractagreement.ContractAgreementService;
+import org.eclipse.edc.connector.spi.contractdefinition.ContractDefinitionService;
+import org.eclipse.edc.connector.spi.contractnegotiation.ContractNegotiationService;
+import org.eclipse.edc.connector.spi.policydefinition.PolicyDefinitionService;
+import org.eclipse.edc.connector.spi.transferprocess.TransferProcessService;
+import org.eclipse.edc.junit.extensions.EdcExtension;
+import org.eclipse.edc.spi.asset.DataAddressResolver;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.eclipse.edc.junit.testfixtures.TestUtils.getFreePort;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.mockito.Mockito.mock;
+
+//todo: uncomment after test refactoring
+//@ApiTest
```

Comment:
We should avoid adding TODOs to the source code. Since this is just a workaround for now, I think we can delete these lines or?

```suggestion
```

Reply:
We'll have the refactoring for tests sooner than we upgrade to the next EDC version, so I'd leave them in for the time being.